src/models/Company.js

```diff
@@ -23,6 +23,10 @@ const CompanySchema = new Schema({
         type: Boolean,
         default: false
     },
+    isBlocked: {
```

Comment:
I think we should have an internal `blockReason`, similar to the `adminReason` on disabled offers. Please make sure this field is hidden for anyone which is not an admin or god. #154 might help, since it's doing a similar thing, and might have some utils which can be reused here